src/options/options.html

```diff
@@ -144,6 +144,13 @@ <h2 data-i18n="__MSG_subtitleEntryPreview__">Entry design</h2>
 						<span data-i18n="__MSG_optionMenuShowReadableTextDescr__" class="line indent helper-text">In addition to the examples of the transformation, this also shows the name of transformation as text.</span>
 					</li>
 
+					<li>
+						<div class="line">
+							<input class="setting save-on-change" type="checkbox" id="nested" data-optiongroup="unicodeFont" name="nested">
+							<label for="nested">Create nested context menu</label>
```

Comment:
```suggestion
							<input class="setting save-on-change" type="checkbox" id="nested" data-optiongroup="unicodeFont" name="nested">
							<label for="nested" data-i18n="__MSG_optionMenuNested__">Create nested context menu</label>
```

Then just add `optionMenuNested` to the language JSON and it is localized.

Reply:
OK, I figured you would want to change this text anyway when you localized it to German.